tools/copy-aos-process.mjs

```diff
@@ -0,0 +1,62 @@
+import { exec } from 'node:child_process';
+import { promisify } from 'node:util';
+import { rm, rename, copyFile } from 'node:fs/promises';
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'url';
+import yaml from 'js-yaml';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const execAsync = promisify(exec);
+
+const configContent = fs.readFileSync(
+  path.join(__dirname, '../ao-build-config.yml'),
+  'utf-8',
+);
+
+const config = yaml.load(configContent);
+
+const repoUrl = 'https://github.com/permaweb/aos.git';
+const commitHash = config.aos_git_hash;
+const tempRepoDir = path.resolve('temp-repo');
+const processTargetDir = path.resolve('tools/fixtures/aos-process');
+const configSourceFile = path.resolve('ao-build-config.yml');
+const configDestFile = path.join(processTargetDir, 'config.yml');
+
+async function copyAosProcess() {
+  try {
+    // Step 1: Remove existing `aos-process` directory
+    console.log(`Removing existing directory: ${processTargetDir}`);
+    await rm(processTargetDir, { recursive: true, force: true });
+
+    // Step 2: Clone the repository into a temporary directory
+    console.log(`Cloning repository: ${repoUrl}`);
+    await execAsync(`git clone ${repoUrl} ${tempRepoDir}`);
```

Comment:
I think you can speed this section up by:
- making a shallow clone: `git clone --depth 1 ${repoUrl} ${tempRepoDir}`
- fetching only the commit hash we need: `git fetch --depth 1 origin ${commitHash}`
- checking out that hash: `git checkout ${commitHash}`

Might even be able to do it in fewer commands if you had a tag for the commit hash 🤔

Reply:
Gonna leave this for the moment in favor of moving this forward.